src/utils/titleHelper.ts

```diff
@@ -0,0 +1,26 @@
+import { PageTitles } from 'common/constants';
+
+import { ServiceContainerData } from 'hooks/useServiceContainer';
+
```

Comment:
In the end single implementation like this could be enough, it also pluralize more correctly:

```tsx
// usage - detail
generateTitle({
   serviceContainer: serviceContainerProject,
   entity: 'Project',
})


// usage - edit
generateTitle({
   serviceContainer: serviceContainerProject,
   entity: 'Project',
   pageType: 'edit'
})


// implementation:
```
![image](https://user-images.githubusercontent.com/17566891/220692928-780833aa-274d-4543-895d-0baaf725a439.png)